package.json

```diff
@@ -81,7 +76,7 @@
     "sass-lint": "^1.5.1",
     "sass-loader": "^3.1.2",
     "sasslint-loader": "0.0.1",
-    "style-loader": "^0.13.0",
+    "style-loader": "magalhas/style-loader#6ffa6df",
```

Comment:
Thought I'd mentioned it in the PR description, but apparently I forgot.  I'll update that.

style-loader doesn't work in the Node environment, but there is an open PR that allows it to.  The referenced repo/commit is the origin of the PR.  I'm subscribed to the PR, so I'll know when it gets merged.